inst/htmljs/animint.js

```diff
@@ -1648,7 +1648,7 @@ var animint = function (to_select, json_file) {
     }
     
     // selecting points based on legend
-    d3.select("#plot").selectAll("#legend").selectAll("tr")
+    d3.selectAll("#legend").selectAll("tr")
```

Comment:
I think the problem with rendering multiple animints on the same web page may be related to the fact that you are currently using `d3.selectAll` here. I think it would make more sense to use `element.selectAll` or `svg.selectAll` (depending on if you want to select all legend entries in the entire animint or plot). Remember we have these definitions:

```javascript
  var element = d3.select(to_select);

    var svg = tdLeft.append("svg")
      .attr("id", p_name)
      .attr("height", p_info.options.height)
      .attr("width", p_info.options.width);
```